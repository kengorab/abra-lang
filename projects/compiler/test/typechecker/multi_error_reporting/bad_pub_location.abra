func foo() {
  pub val x = 123

  val y = x + []
}