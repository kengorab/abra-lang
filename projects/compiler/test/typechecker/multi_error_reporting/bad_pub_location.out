Error at %TEST_DIR%/typechecker/multi_error_reporting/bad_pub_location.abra:2:3
Invalid visibility modifier
  |    pub val x = 123
       ^
Exported values may only appear at the top level scope in a module

Error at %TEST_DIR%/typechecker/multi_error_reporting/bad_pub_location.abra:4:13
Illegal operator
  |    val y = x + []
                 ^
No operator '+' exists between types Int and <unknown>[]
